test/test_fakeredis.py

```diff
@@ -1597,12 +1597,12 @@ def test_hsetnx(r):
     assert r.hget('foo', 'newkey') == b'v1'
 
 
-def test_hmsetset_empty_raises_error(r):
+def test_hmset_empty_raises_error(r):
```

Comment:
Changes in this file aren't really related - just a typo/search-replace error that I spotted.